lib/go-rfc/http.go

```diff
@@ -36,6 +36,8 @@ const (
 	ContentDisposition = "Content-Disposition" // RFC6266
 	ContentEncoding    = "Content-Encoding"    // RFC7231§3.1.2.2
 	ContentType        = "Content-Type"        // RFC7231§3.1.1.5
+	PermissionsPolicy  = "Permissions-Policy"  // W3C "Permissions Policy"
+	Server             = "Server"              // RFC7231§7.4.2
```

Comment:
Were you planning to do something with `rfc.Server`?

Reply:
I was, but then I decided I should do it in a different PR. But I didn't want to look up the RFC number and section again, and it's not hurting anything so I left it in.

Reply:
Do you want me to take it out? I don't really feel strongly about it. It's an artifact of laziness.

Reply:
Your reason for leaving it in made sense to me, leaving it sounds fine